### PR TITLE
Ds 1396 add register user endpoint

### PR DESCRIPTION
**Pull request by bahulneel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Darkspark Example using Express
+
 This repository illustrates using Darkspark with Express on nodejs.
 
 ## How to Use
+
 1. Create an account on [Darkspark](https://darkspark.io)
 2. Skip the onboarding dialog and click "Take me to API keys". Or just go directly to [API Keys](https://darkspark.io/account/apikeys).
```
Comment (robbobby) [inline README.md:8]: There is a pre step to this to navigate to the users default workspace.

Comment (bahulneel) [reply README.md:8]: Can you suggest some text?

Comment (neil-gall) [inline README.md:8]:
```suggestion
2. Skip the onboarding dialog and navigate to Account / [API Keys](https://darkspark.io/account/apikeys).
```

**Review by robbobby — APPROVED** [review]

File: index.mjs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        Users,
+        RegisterUser
+    },
+    consumesMiddleware: {
+        'application/json': bodyParser.json(),
+        'text/text': bodyParser.text()
+    }
+});
+
+// console.log(apiDoc['paths']['/users']['post']);
```
Comment (rujames) [inline index.mjs:52]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -13,15 +15,17 @@ This repository illustrates using Darkspark with Express on nodejs.
 6. Run `npm start`
```
Comment (robbobby) [inline README.md:15]:
```suggestion
6. Run `npm install`
7. Run `npm start`
```

**Review by robbobby — CHANGES_REQUESTED** [review]

File: .prettierrc [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "overrides": [
+    {
+      "files": ["*.js", "*.ts", "*.mjs"],
+      "options": {
+        "tabWidth": 4
+      }
+    }
+  ]
+}
```
Comment (rujames) [inline .prettierrc:10]: The files in this PR haven't been formatted in accordance with this Prettier configuration.  This should only be included if we intend to adhere to it.

File: index.mjs [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+const { darkspark } = darksparkPlug;
+const app = express();
+const port = 3000;
+
+const API_KEY = process.env.DARKSPARK_API_KEY;
+
+if (!API_KEY) {
+    console.error("Please set DARKSPARK_API_KEY environment variable");
+    console.error("Darkspark Disabled!")
+    // process.exit(1);
```
[the remark this answers is not in the document]
Comment (bahulneel) [reply index.mjs:23]: I didn't like that behaviour, I may want to check out the sample so before I get an API key

[the remark this answers is not in the document]
Comment (rujames) [reply index.mjs:23]:
```suggestion
```

**Review by neil-gall — APPROVED** [review]

**Pull request merged**